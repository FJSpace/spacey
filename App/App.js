import React from 'react';
import { View, Text } from 'react-native';
import { StackNavigator } from 'react-navigation';
import HomePage from './Page/HomePage.js'
import DetailPage from './Page/DetailPage.js'


const RootStack = StackNavigator({
  Home: { screen: HomePage },
  Detail: { screen: DetailPage },
},
{
  initialRouteName: 'Home',
  navigationOptions:{
    headerStyle:{
      backgroundColor: '#B7B9B8'
    },
    headerTintColor: '#0C3F7D'
  }
}
);

export default class App extends React.Component {

  render() {
    return <RootStack />;
  }

}
