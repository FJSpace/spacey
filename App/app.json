{
  "expo": {
    "sdkVersion": "25.0.0",
    "splash": {
      "image": "./assets/images/splash.png",
      "resizeMode": "cover"
    },
    "packagerOpts": {
      "assetExts": ["ttf"]
    }
  }
}
