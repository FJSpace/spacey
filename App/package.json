{
  "name": "Code",
  "version": "0.1.0",
  "private": true,
  "devDependencies": {
    "enzyme": "^3.3.0",
    "enzyme-adapter-react-16": "^1.1.1",
    "enzyme-to-json": "^3.3.3",
    "identity-obj-proxy": "^3.0.0",
    "jest-expo": "25.0.0",
    "react-addons-test-utils": "^15.6.2",
    "react-dom": "^16.3.2",
    "react-native-scripts": "1.11.1",
    "react-test-renderer": "16.2.0"
  },
  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
  "scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "jest --u --coverage"
  },
  "jest": {
    "setupTestFrameworkScriptFile": "./test-setup.js",
    "preset": "jest-expo",
    "snapshotSerializers": [
      "./node_modules/enzyme-to-json/serializer"
    ],
    "moduleNameMapper": {
      ".+\\.(css|styl|less|sass|scss|png|jpg|ttf|woff|woff2)$": "identity-obj-proxy"
    }
  },
  "dependencies": {
    "expo": "^25.0.0",
    "react": "16.2.0",
    "react-native": "0.52.0",
    "react-native-elements": "^0.19.1",
    "react-native-sortable-listview": "^0.2.8",
    "react-native-textinput-effects": "^0.4.2",
    "react-native-vector-icons": "^4.6.0",
    "react-navigation": "^1.5.11"
  }
}
