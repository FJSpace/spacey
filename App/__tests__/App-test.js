import React from 'react';
import { shallow, mount, render } from 'enzyme';
import App from '../App';
import renderer from 'react-test-renderer';


it('renders without crashing', () => {
  const wrapper = shallow(
      <App />
    );

    expect(wrapper).toMatchSnapshot();
});
