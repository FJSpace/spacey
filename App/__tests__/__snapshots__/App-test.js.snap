// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<AppLoading
  onError={[Function]}
  onFinish={[Function]}
  startAsync={[Function]}
/>
`;
